README.md

```diff
@@ -66,13 +68,15 @@ Or update submodules at any time with:
 git submodule update --init --recursive
 ```
 
+NOTE: The previous steps does not apply to Windows users, because no command-line method allow user to clone/get a github repository.
+
```

Comment:
https://desktop.github.com/

Reply:
I understood that was useless so I didn't add anything arround this. Is it ok?